docs/en/docs/preqequisites.md

```diff
@@ -0,0 +1,6 @@
+# Prerequisites
```

Comment:
Having a separate page with prerequisites looks neat, good idea!

Reply:
Thank you!